fixtures/dev/users.yaml

```diff
@@ -0,0 +1,508 @@
+    fullyEquipped: '<boolean(50)>'
+  User.UL-05.{1..10}:
+    firstName: '<fr_FR:firstName()>'
+    lastName: '<fr_FR:lastName()>'
+    organization: '@Organization.UL-05'
+    identificationNumber: '<identificationNumber()>'
+    emailAddress (unique): '<fr_FR:safeEmail()>'
+    phoneNumber: '<fr_FR:phoneNumber()>'
+    birthday: '1990-01-01'
+    occupation: '<randomElement(["Pharmacien", "Pompier", "Ambulancier.e", "Logisticien", "Infirmier.e"])>'
+    organizationOccupation: Secouriste
+    skillSet: '<skillSet(<{app.available_skills}>)>'
+    vulnerable: '<boolean(50)>'
+    fullyEquipped: '<boolean(50)>'
+  User.UL-06.{1..10}:
+    firstName: '<fr_FR:firstName()>'
+    lastName: '<fr_FR:lastName()>'
+    organization: '@Organization.UL-06'
+    identificationNumber: '<identificationNumber()>'
+    emailAddress (unique): '<fr_FR:safeEmail()>'
+    phoneNumber: '<fr_FR:phoneNumber()>'
+    birthday: '1990-01-01'
+    occupation: '<randomElement(["Pharmacien", "Pompier", "Ambulancier.e", "Logisticien", "Infirmier.e"])>'
+    organizationOccupation: Secouriste
+    skillSet: '<skillSet(<{app.available_skills}>)>'
+    vulnerable: '<boolean(50)>'
+    fullyEquipped: '<boolean(50)>'
+  User.UL-07.{1..10}:
+    firstName: '<fr_FR:firstName()>'
+    lastName: '<fr_FR:lastName()>'
+    organization: '@Organization.UL-07'
+    identificationNumber: '<identificationNumber()>'
+    emailAddress (unique): '<fr_FR:safeEmail()>'
+    phoneNumber: '<fr_FR:phoneNumber()>'
+    birthday: '1990-01-01'
+    occupation: '<randomElement(["Pharmacien", "Pompier", "Ambulancier.e", "Logisticien", "Infirmier.e"])>'
+    organizationOccupation: Secouriste
+    skillSet: '<skillSet(<{app.available_skills}>)>'
+    vulnerable: '<boolean(50)>'
+    fullyEquipped: '<boolean(50)>'
+  User.UL-08.{1..10}:
+    firstName: '<fr_FR:firstName()>'
+    lastName: '<fr_FR:lastName()>'
+    organization: '@Organization.UL-08'
+    identificationNumber: '<identificationNumber()>'
+    emailAddress (unique): '<fr_FR:safeEmail()>'
+    phoneNumber: '<fr_FR:phoneNumber()>'
+    birthday: '1990-01-01'
+    occupation: '<randomElement(["Pharmacien", "Pompier", "Ambulancier.e", "Logisticien", "Infirmier.e"])>'
+    organizationOccupation: Secouriste
+    skillSet: '<skillSet(<{app.available_skills}>)>'
+    vulnerable: '<boolean(50)>'
+    fullyEquipped: '<boolean(50)>'
+  User.UL-09.{1..10}:
+    firstName: '<fr_FR:firstName()>'
+    lastName: '<fr_FR:lastName()>'
+    organization: '@Organization.UL-09'
+    identificationNumber: '<identificationNumber()>'
+    emailAddress (unique): '<fr_FR:safeEmail()>'
+    phoneNumber: '<fr_FR:phoneNumber()>'
+    birthday: '1990-01-01'
+    occupation: '<randomElement(["Pharmacien", "Pompier", "Ambulancier.e", "Logisticien", "Infirmier.e"])>'
+    organizationOccupation: Secouriste
+    skillSet: '<skillSet(<{app.available_skills}>)>'
+    vulnerable: '<boolean(50)>'
+    fullyEquipped: '<boolean(50)>'
+  User.UL-11.{1..10}:
```

Comment:
why not something like `User.UL-{1..20}.{1..10}`, or ` organization: '@Organization.UL-<current()%20>`: ?